wearApp/src/test/kotlin/dev/johnoreilly/confetti/wear/screenshots/BaseScreenshotTest.kt

```diff
@@ -0,0 +1,248 @@
+package dev.johnoreilly.confetti.wear.screenshots
+
+import android.app.Application
+import android.content.Context
+import android.content.res.Resources
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.BoxScope
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.test.junit4.ComposeContentTestRule
+import androidx.test.core.app.ApplicationProvider
+import androidx.wear.compose.material.PositionIndicator
+import androidx.wear.compose.material.Scaffold
+import androidx.wear.compose.material.TimeText
+import androidx.wear.compose.material.scrollAway
+import com.google.android.horologist.annotations.ExperimentalHorologistApi
+import com.google.android.horologist.compose.layout.ScalingLazyColumnDefaults
+import com.google.android.horologist.compose.layout.ScalingLazyColumnState
+import com.google.android.horologist.compose.tools.coil.FakeImageLoader
+import dev.johnoreilly.confetti.screenshot.RNGScreenshotTestRule
+import dev.johnoreilly.confetti.wear.FixedTimeSource
+import dev.johnoreilly.confetti.wear.app.KoinTestApp
+import dev.johnoreilly.confetti.wear.preview.TestFixtures
+import dev.johnoreilly.confetti.wear.proto.Theme
+import dev.johnoreilly.confetti.wear.ui.ConfettiTheme
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import okio.FileSystem
+import okio.Path
+import org.junit.Rule
+import org.junit.runner.RunWith
+import org.koin.core.context.stopKoin
+import org.koin.test.KoinTest
+import org.robolectric.ParameterizedRobolectricTestRunner
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.GraphicsMode
+import kotlin.test.AfterTest
+
+@RunWith(RobolectricTestRunner::class)
+@Config(
+    application = KoinTestApp::class,
+    sdk = [30],
+    qualifiers = "w221dp-h221dp-small-notlong-round-watch-xhdpi-keyshidden-nonav"
+)
+@GraphicsMode(GraphicsMode.Mode.NATIVE)
+abstract class BaseScreenshotTest(
```

Comment:
Not quite sure on the abstraction here. One drawback at least is that you have to configure the tolerance for each class not for each test as the other API had.  Wdyt @yschimke ? 🤔

Reply:
I wouldn't overthink it in a first version.  Get something working then iterate.